src/game.cpp

```diff
@@ -92,8 +93,10 @@ bool is_valid_in_w_terrain(int x, int y)
 
 // This is the main game set-up process.
 game::game() :
+    map_ptr( new map() ),
     new_game(false),
     uquit(QUIT_NO),
+    m( *map_ptr.get() ),
```

Comment:
`.get()` is not really required, `unique_ptr` has a `operator*` which returns the desired reference.

cata_tiles.cpp also needs the map.h include.